.github/workflows/backend_deploy.yml

```diff
@@ -26,14 +26,21 @@ jobs:
         env:
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-      - name: 'Build Docker image'
-        run: |
-          docker build -t ${{ env.serviceName }} .
+      - name: 'Set up JDK 17'
```

Comment:
Čo keby si to dal inde než "backend_deploy"? Nehodí sa mi to tu. Workflow má spraviť build a deploy. Testy by som tu nehľadal. Okrem toho sa spustie len a len v prípade master branch. Ked už testy, tak všade - hlavne PR, tam by to mala byť podmienka pred review.

Reply:
Ešte doplním ukážku, kde som veľmi spokojný: https://github.com/cesko-digital/loono/actions/runs/1837277651/workflow

Aktuálne kontrola formátu a analýza lintov perfektne funguje pri PR. Bez toho sa nejde na review. Chcel by som niečo podobné docieliť aj s testami.

Reply:
Tak to nejsou testy. To je pro situaci, kdy build pri deployi spadne a ty potrebujes znat vysledky testu. Tzn. nepoustime nic navic, jen publish vysledku pro lepsi debugging.

Reply:
jj, mate pravdu, vyrevertoval jsem puvodni zmeny, zakladam zvlastni worfklow na build PR a masteru, v ramci ktereho budeme mit i test reporty